java/WORKSPACE

```diff
@@ -0,0 +1,26 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+RULES_JVM_EXTERNAL_TAG = "3.3"
+RULES_JVM_EXTERNAL_SHA = "d85951a92c0908c80bd8551002d66cb23c3434409c814179c0ff026b53544dab"
+
+http_archive(
+    name = "rules_jvm_external",
+    strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_TAG,
+    sha256 = RULES_JVM_EXTERNAL_SHA,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
+)
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+maven_install(
+    artifacts = [
+        "com.github.javaparser:javaparser-core:3.23.1",
+        "com.github.javaparser:javaparser-symbol-solver-core:3.23.1",
+        "org.json:json:20190722",
+        "junit:junit:3.8.1",
+    ],
+    repositories = [
+        "https://jcenter.bintray.com/",
```

Comment:
AFAIK bintray has been sunsetted, so you can remove this entry. https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/